Assets/MixedRealityToolkit.SDK/Features/Input/Handlers/ControllerPoseSynchronizer.cs

```diff
@@ -12,6 +12,8 @@ namespace Microsoft.MixedReality.Toolkit.Input
     /// </summary>
     public class ControllerPoseSynchronizer : InputSystemGlobalListener, IMixedRealityControllerPoseSynchronizer
     {
+        protected bool hasSource = false;
```

Comment:
Can we use `IsTracked` for this?

Reply:
Or maybe `TrackingState`?

Reply:
Following from Julia's suggestion, we should probably use `enabled`